ql/src/java/org/apache/hadoop/hive/ql/optimizer/calcite/cost/HiveVolcanoPlanner.java

```diff
@@ -85,7 +85,6 @@ public void registerClass(RelNode node) {
       addRule(HiveDruidRules.FILTER_PROJECT_TRANSPOSE);
       addRule(HiveDruidRules.SORT_PROJECT_TRANSPOSE);
       addRule(HiveDruidRules.SORT);
-      addRule(HiveDruidRules.PROJECT_SORT_TRANSPOSE);
```

Comment:
Double check this

Reply:
This rule is useless, and has been removed in 1.24. [CALCITE-4023](https://issues.apache.org/jira/browse/CALCITE-4023)